pkg/debug/ssh.sh

```diff
@@ -7,6 +7,43 @@ echo -1 > /proc/sys/kernel/perf_event_paranoid
 KEYS=$(find /etc/ssh -name 'ssh_host_*_key')
 [ -z "$KEYS" ] && ssh-keygen -A >/dev/null 2>/dev/null
 
+# Background monitoring non-ssh related tasks
+monitor_file_and_execute_tasks() {
+    # edgeview request to generate collect-info and remove tar.gz file
+    # the protocol is for edgeview to create a file
+    # /run/edgeview/edgeview-request-collect-info, and debug container
+    # run 'collect-info.sh' to generate the tar.gz file, when the job is
+    # done, remove the file /run/edgeview/edgeview-request-collect-info.
+    # edgeview can also request to remove the tar.gz file by creating
+    # /run/edgeview/edgeview-request-remove-tar.gz file, and debug container
+    # will get the file name and remove the generated tar.gz file.
+    #
+    # if addition tasks are needed, this can monitor multiple directories
+    # and perform different actions by debug container in the future.
+    while inotifywait -e create /run/edgeview; do
+        if [ -f "/run/edgeview/edgeview-request-collect-info" ]; then
+            echo "edgeview request to run collect-info..." > /dev/kmsg
+            # the newlog part of the collection, only collect last 10 days
+            /usr/bin/collect-info.sh -t 10
```

Comment:
Where will the output (stdout, stderr) of `collect-info.sh` be written to?

Reply:
will get collected through device newlogs